### DISCUSSION TITLE
Помогите с запуском сервера

### DISCUSSION THREAD

**Discussion by nikolaykotikov** [Arkady23/V380-Web#68] [Q&A]
Доброго времени суток!
Имеется камера с параметрами:
AppEV2W_VA3_V1.3.4.3_20230914
HwAWT3610E_XHR_E_V1.0_WF_20230807
Копирую файлы на носитель (FAT32, exFAT), сервер telnet не запускается. Подключиться не могу
Помогите решить проблему

**Comment by Arkady23** [comment]
Появился новый способ установки веб на камеру. В ближайшее время опишу на главной странице.
